src/mpi/romio/test/aggregation1.c

```diff
@@ -6,7 +6,7 @@
 /* Test case from John Bent (ROMIO req #835)
  * Aggregation code was not handling certain access patterns when collective
  * buffering forced */
-#define _XOPEN_SOURCE 500       /* strdup not in string.h otherwise */
+#define _XOPEN_SOURCE 600       /* strdup not in string.h otherwise */
```

Comment:
This is probably an artifact of Solaris testing. Can we just remove the define? As far as I can tell, `strdup` is available on our test systems.

Reply:
Oh, sorry. I just re-read the commit message. I guess it is still needed.

Reply:
Actually I think it's still fine to remove unless we encounter a system where `strdup` is not available by default.

Reply:
@raffenet I am fine with removing it. Let me know which way you prefer.

Reply:
I'll just approve it as it is since it's confirmed working and no need to rock the boat.